CMake/External_VTK.cmake

```diff
@@ -54,13 +54,31 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # Qt
-add_package_dependency(
-  PACKAGE VTK
-  PACKAGE_DEPENDENCY Qt
-  PACKAGE_DEPENDENCY_ALIAS Qt4
-)
-if(QT_QMAKE_EXECUTABLE)
-  set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+if(Qt_version_major EQUAL 4)
+  add_package_dependency(
+    PACKAGE VTK
+    PACKAGE_DEPENDENCY Qt
+    PACKAGE_DEPENDENCY_ALIAS Qt4
+  )
+  if(QT_QMAKE_EXECUTABLE)
+    set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+    set(vtk_cmake_args ${vtk_cmake_args}
+      -DVTK_RENDERING_BACKEND:STRING=OpenGL2
+      -DQT_QMAKE_EXECUTABLE:PATH=${QT_QMAKE_EXECUTABLE}
+    )
+  endif()
+else()
+  add_package_dependency(
+    PACKAGE VTK
+    PACKAGE_DEPENDENCY Qt
+    PACKAGE_DEPENDENCY_ALIAS Qt5
+    PACKAGE_DEPENDENCY_COMPONENTS
+      Core Gui Widgets OpenGL Designer UiPlugin
+  )
+  set(vtk_cmake_args ${vtk_cmake_args}
+    -DVTK_RENDERING_BACKEND:STRING=OpenGL
+    -DQt5_DIR:PATH=${Qt5_DIR}
```

Comment:
The default for `VTK_QT_VERSION` is 5 but doesn't hurt to reset it in the case of reconfiguration.